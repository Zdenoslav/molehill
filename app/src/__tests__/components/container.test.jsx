import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer'
import 'jest-styled-components';

import Container, {StyledContainer} from 'app/components/container';
import Theme from 'app/components/styleguide/theme';

describe('Container', () => {

  it('should render Container', () => {
    const wrapper = shallow(<Container />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render StyledContainer', () => {
    const wrapper = renderer.create(<StyledContainer theme={Theme} />).toJSON();
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toHaveStyleRule('background-color', Theme.background);
  });

});
