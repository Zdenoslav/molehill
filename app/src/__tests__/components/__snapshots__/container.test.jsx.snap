// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render Container 1`] = `<Styled(Styled(styled.div)) />`;

exports[`Container should render StyledContainer 1`] = `
.c0 {
  width: auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background-color: #fefefe;
  box-shadow: unset;
  padding: 16px;
  height: auto;
  border-radius: 0;
}

<div
  className="c0"
/>
`;
